sdk/storagesync/Azure.ResourceManager.StorageSync/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
```

Comment:
please specify the release date in the changelog, so that we know when to release it.